### ISSUE TITLE
Edit fields are not resizable

### ISSUE THREAD

**Issue by zalintyre** [echoma/zkui#14]
Altough the value field is pretending to be resizable, it isn't. It's a pain to edit larger text files in there.

**Comment by echoma** [comment]
the edit dialog can be resized by the drag the right-bottom corner of the text area (not the dialog).

**Comment by zalintyre** [comment]
Ah, indeed. I didn't notice this, because the cursor doesn't change at that point.

Thanks! A changing cursor would be a nice feature, though.

**Comment by echoma** [comment]
You are right, a resize cursor on the dialog corner would be nice. It seems that the modal dialog component of bootstrap (i am using it to implement the UI of zkui) does not support resizing. zkui realy need some improments..
